src/BibleWell.Aquifer.Data/SqliteAquiferService.cs

```diff
@@ -1,14 +1,49 @@
-﻿namespace BibleWell.Aquifer.Data;
+﻿using BibleWell.Aquifer.Data.DbModels;
+using BibleWell.Storage;
+
+namespace BibleWell.Aquifer.Data;
 
 public sealed class SqliteAquiferService : IReadWriteAquiferService
 {
-    public Task<ResourceContent?> GetResourceContentAsync(int contentId)
+    private readonly ResourceContentRepository _resourceContentRepository;
+
+    public SqliteAquiferService(IStorageService storageService)
+    {
+        var dbManager = new SqliteDbManager(storageService);
+        _resourceContentRepository = new ResourceContentRepository(dbManager);
+    }
+
+// SQLite is not async, we have a common interface that does interact with an async API.
+// This might change as we develop the application further. For now, we are telling the comiler to ignore this.
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+    public async Task<ResourceContent?> GetResourceContentAsync(int id)
+
+    {
+        var resourceContent = _resourceContentRepository.GetById(id);
+
+        if (resourceContent == null)
+        {
+            return null;
+        }
+        return MapToResourceContent(resourceContent);
```

Comment:
I suggest updating the mapper to handle nulls which will prevent all callers from needing this logic.